src/server/tiered_storage.cc

```diff
@@ -171,6 +171,16 @@ util::fb2::Future<string> TieredStorage::Read(DbIndex dbid, string_view key,
   return future;
 }
 
+void TieredStorage::Read(DbIndex dbid, std::string_view key, const PrimeValue& value,
+                         std::function<void(const std::string&)> readf) {
+  DCHECK(value.IsExternal());
+  auto cb = [readf = std::move(readf)](string* value) {
+    readf(*value);
+    return false;
```

Comment:
add a comment what false means

Reply:
I have a comment on the callback type what false means, so I don't have to repeat it everywhere 🙂

Reply:
you have a typo there: `Returns whether the value was modified`. you mean `returns true if ....`

Reply:
That's the same meaning, no? 🌹